package.json

```diff
@@ -65,6 +70,7 @@
         "highcharts": "^10.2.0",
         "lodash": "^4.17.21",
         "mathjs": "^9.4.2",
+        "multi-calendar-dates": "1.0.0-alpha.12",
```

Comment:
Minor comment: I expected this to have the `@dhis2` namespace.

Reply:
Well spotted @edoardo. Yeah, the `name` field in the [`package.json`](https://github.com/dhis2/multi-calendar-dates/blob/beta/package.json#L2) in should be updated to include the `@dhis2` namespace, like [here](https://github.com/dhis2/app-runtime/blob/master/runtime/package.json#L2).